app/components/Header.tsx

```diff
@@ -45,6 +45,7 @@ export default function Header() {
             alignItems="center"
             justifyContent={{ base: "flex-start", lg: "flex-end" }}
             gap={{ base: "0", lg: "24px" }}
+            overflow="hidden"
```

Comment:
This hides already "hidden" "Home" and "Request a quote" buttons on screens < 992px. Otherwise, even though "Home" and "Request a quote" buttons are hidden on screens < 992px, the space for them is still "reserved" and shows when you scroll.